static/js/theme-map/Maps/Providers/Mapbox.js

```diff
@@ -175,22 +183,38 @@ const yextAPIKey = 'pk.eyJ1IjoieWV4dCIsImEiOiJqNzVybUhnIn0.hTOO5A1yqfpN42-_z_GuL
  * @see {MapProvider~loadFunction}
  */
 function load(resolve, reject, apiKey, {
-  version = 'v1.6.1'
+  version = 'v1.13.0'
```

Comment:
Is there a 1.13.1 we can use? I noticed one exists here https://github.com/mapbox/mapbox-gl-js/releases?after=v2.2.0 but not sure if that's the same thing

Reply:
oh I was looking at here: https://github.com/mapbox/mapbox-gl-js/blob/main/CHANGELOG.md. looks different because the bug fix from your link is mentioned in v2.0.1 from my link instead

Reply:
Ah okay would it be okay for you to try to see if 1.13.1 works? I imagine they just forgot to update their changelog since their npm also has a 1.13.1

Reply:
yea, will do!